addons/website/static/src/builder/plugins/theme/theme_advanced_option.xml

```diff
@@ -46,7 +46,7 @@
             <div class="o_we_gray_preview d-flex w-100">
 	            <t t-foreach="[0,1,2,3,4,5,6,7,8]" t-as="i" t-key="i">
 	                <t t-set="grayCode" t-value="(9 - i) * 100"/>
-	                <span t-attf-title="Gray #{grayCode}"
+	                <span t-att-title="getGrayTitle(grayCode)"
```

Comment:
`t-attf-` are normally translated: if not, that's the bug that should be fixed here